library/src/main/java/com/getstream/sdk/chat/utils/CaptureController.java

```diff
@@ -0,0 +1,75 @@
+package com.getstream.sdk.chat.utils;
+
+import android.annotation.SuppressLint;
+import android.content.ContentValues;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+
+import androidx.annotation.Nullable;
+
+import com.getstream.sdk.chat.R;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class CaptureController {
+
+    private static Uri imageUri, videoUri;
+
+    public static Intent getTakePictureIntent(Context context) {
+        File newFile = new File(getFolderPath(true, context), "IMG_" + getFileName() + ".jpg");
+        imageUri = Uri.fromFile(newFile);
+        Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+        ContentValues values = new ContentValues();
+        values.put(MediaStore.Images.Media.TITLE, "New Picture");
+        values.put(MediaStore.Images.Media.DESCRIPTION, "From your Camera");
+        takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, imageUri);
+        return takePictureIntent;
+    }
+
+    public static Intent getTakeVideoIntent(Context context) {
+        File newFile = new File(getFolderPath(false, context), "VID_" + getFileName() + ".mp4");
+        videoUri = Uri.fromFile(newFile);
+        Intent takeVideoIntent = new Intent(MediaStore.ACTION_VIDEO_CAPTURE);
+        ContentValues values = new ContentValues();
+        values.put(MediaStore.Video.Media.TITLE, "New Video");
+        values.put(MediaStore.Video.Media.DESCRIPTION, "From your Camera");
+        takeVideoIntent.putExtra(MediaStore.EXTRA_OUTPUT, videoUri);
+        return takeVideoIntent;
+    }
+
+    @SuppressLint("SimpleDateFormat")
+    private static String getFileName() {
+        return new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date().getTime());
+    }
+
+    private static String getFolderPath(boolean isImage, Context context) {
+        String appName = Utils.getApplicationName(context);
+        String folderPath = Environment.getExternalStorageDirectory() + "/"
+                + appName + " " + context.getString(isImage ? R.string.stream_image : R.string.stream_video);
```

Comment:
Let's also update `readme`. Developers might ask how and where we're storing pictures and videos.